File: Dataset/Leetcode/train/83/202.c
struct ListNode* XXX(struct ListNode* head){
    struct ListNode *cur = head;
    while (cur!=NULL && cur->next != NULL) {
        if (cur->val == cur->next->val) {
            cur->next = cur->next->next;
        }
        else {
            cur = cur->next;
        }
        
    }
     return head;
}

File: Dataset/Leetcode/train/19/202.java
 class Solution {
    public ListNode XXX(ListNode head, int n) {
        ListNode quick = head,slow = head;
        for(int i=0;i<n;i++) quick = quick.next;
        if(quick==null) return head.next;
        while(quick.next!=null){
            quick = quick.next;
            slow = slow.next;
        }
        slow.next = slow.next.next;
        return head;
    }
}

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }

